src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(element => element.sex === 'm');
+
+  const centuryMenDiedIn = men.filter(element =>
+    century
+      ? Math.ceil(element.died / 100) === century
+      : element);
+
+  const menAge = getAge(centuryMenDiedIn);
+
+  return menAge.reduce((a, b) => a + b, 0) / menAge.length;
```

Comment:
this begs to be in a single helper function.
you're using it twice at the least. Make a helper.